svthalia/concrexit
# Implement registration update deadlines

Closes #3657.



### Summary
Adds an `update_deadline` field that allows updating registrations until after the registration deadline.

NOTE: setting update_deadline before registration_end does nothing. This is also documented in the help text and checked by `clean()`

### How to test

1. Create event with update_deadline set after registration_end, and some fields
      1. Also check having it before -> clean error
1. Register for it
1. Let registration_end expire (or move it ahead)
1. Check that fields can still be edited until update_deadline.
1. Check that afterwards it cannot be anymore.
